apps/crn-server/test/fixtures/research-output.fixtures.ts

```diff
@@ -282,8 +283,11 @@ export const getResearchOutputCreateDataObject =
 
 export const getResearchOutputUpdateDataObject =
   (): ResearchOutputUpdateDataObject => {
-    const { createdBy: _, ...researchOutputCreateDataObject } =
-      getResearchOutputCreateDataObject();
+    const {
+      createdBy: _,
+      addedDate: __,
```

Comment:
similar to my comments above i think this is incorrect - both create and update should allow for modification of addedDate

Reply:
I thought both create and update should not allow anything regarding addedDate.

create because it will set it as current time
update because it cannot be changed

as per ticket AC:
When a user creates a new Research Output the ‘addedDate’ field is populated with the current date, **as expected**